polymer/src/sample-detail.html

```diff
@@ -96,15 +98,6 @@
 
     <div>[[content]]</div>
 
-    <!--
-      shop-network-warning shows a warning message when the items can't be rendered due
-      to network conditions.
-    -->
-    <shop-network-warning
-        hidden$="[[!failure]]"
-        offline="[[offline]]"
-        on-try-reconnect="_tryReconnect"></shop-network-warning>
-
```

Comment:
why did you remove the offline warning?

Reply:
Because I am not adding that component at the moment, for simplicity.